src/classes/isotopedata.h

```diff
@@ -4,10 +4,9 @@
 #pragma once
 
 #include "data/isotopes.h"
-#include "templates/listitem.h"
 
 // sotopeData Definition
```

Comment:
```suggestion
// IsotopeData Definition
```
Not your typo, but nice to fix while we're here!